spotifycli/spotifycli.py

```diff
@@ -14,6 +15,11 @@
 from jeepney.io.blocking import open_dbus_connection
 
 
+class SpotifyCLIException(Exception):
+    """An exception wrapper purely to handle known exceptions nicely"""
+    pass
```

Comment:
This derived exception will give the library a chance to differentiate between "known" exceptions and then "everything else."   If there is a condition that is likely to occur, and can be handled with specific user guidance, then the worker functions should throw one of these SpotifyCLIException instances.  Any other unknown/unhandled exceptions can bubble up the stack through the base Exception class.